src/Altinn.Notifications.Persistence/Configuration/PostgreSQLSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿#nullable disable
+namespace Altinn.Notifications.Persistence.Configuration
+{
+    /// <summary>
+    /// Settings for Postgre database
+    /// </summary>
+    public class PostgreSqlSettings
+    {
+        /// <summary>
+        /// Connection string for the postgre db
+        /// </summary>
+        public string ConnectionString { get; set; }
```

Comment:
We could initialize these two properties to be an empty string. Alternatively, if possible, assign a value that would work for localhost.